examples/api/lib/services/text_input/text_input_control.0.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Actually can you add a test for this file?  Here's an example of testing an example: https://github.com/flutter/flutter/blob/master/examples/api/test/material/dialog/show_dialog.0_test.dart

I think if you just do something simple like run the app, press the "A" button, and expect that an "A" was inserted in to the field, then it will provide a lot of good test coverage for the virtual keyboard feature.

Reply:
Nice, thanks! This was the missing piece to ensure that the whole stack works. :+1: